### DISCUSSION TITLE
Failed to resolve entry for package "..\node_modules\@types\simple-peer". The package may have incorrect main/module/exports specified in its package.json

### DISCUSSION THREAD

**Discussion by SupertigerDev** [fwouts/previewjs#2293] [Q&A]
![image](https://github.com/fwouts/previewjs/assets/8193841/5f31667e-f92c-4d14-a2e7-657bd06c5c64)

Hello, i saw this awesome extension, using it with solidjs, but im getting this error for some reason. I have `@types\simple-peer` installed as a dev dependency.

**Comment by fwouts** [comment]
Sorry for the delay @SupertigerDev. Do you have a code sample or repo I could look at? Preview.js shouldn't be looking at `@types` packages at all, so this is a little funny. How do you import `simple-peer` in your code?

**Comment by SupertigerDev** [comment]
Here's reproduction: https://github.com/SupertigerDev/solid-simple-peer-preview-js


And here's more information: 
I'm using a fork of simple-peer (@thaunknown/simple-peer) which itself does not have types. so im using the official types (@types/simple-peer) and by using this option in tsconfig:

```json
"paths": {
   "@thaunknown/simple-peer": ["./node_modules/@types/simple-peer"]
}
 ```
 
 I'm asuming theres a problem with this, although it works fine, just doesnt work in previewjs :(
